src/daemon/collectd.c

```diff
@@ -457,9 +457,7 @@ int notify_systemd (void)
     }
     NOTICE ("Systemd detected, trying to signal readyness.");
 
-    unsetenv ("NOTIFY_SOCKET");
-
-    fd = socket (AF_UNIX, SOCK_DGRAM, /* protocol = */ 0);
+    fd = socket (AF_UNIX, SOCK_DGRAM|SOCK_CLOEXEC, /* protocol = */ 0);
```

Comment:
This fails on EPEL5 (both i386 and x86-64) with:

```
collectd.c:460: error: 'SOCK_CLOEXEC' undeclared (first use in this function)
```

Annoying but I guess we need to `#ifdef` around this …